### PR TITLE
[ADD] Adding alias for rgrep and function for fetch pr

### PR DESCRIPTION
**Pull request by JesusZapata**
Fix https://github.com/Vauxoo/docker-odoo-image/issues/184

### CODE REVIEW

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -235,11 +235,22 @@ git_ps1_style(){
...
+git_fetch_pr() {
+    REMOTE=$1
+    shift 1
+    git fetch -p $REMOTE +refs/pull/*/head:refs/pull/$REMOTE/*
+}
 EOF
 
+# Load .container.profile
+if [ ! -f "~/.container.profile" ]; then
+sources ~/.container.profile
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: `s/sources/    source/g`

File: odoo-shippable/scripts/build-image.sh [hunk 2]
```diff
@@ -235,11 +235,22 @@ git_ps1_style(){
 PS1=$UserMachine$Color_Off$PathShort\$\\n"\$(git_ps1_style)"$Color_Off\$" "
 EOF
 
-# Add alias for multitail and configure
+# Add alias and function
 cat >> /etc/bash.bashrc << EOF
 alias tail2="multitail -cS odoo"
+alias rgrep="rgrep -n"
+git_fetch_pr() {
+    REMOTE=$1
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: Could you receive a second parameter for the PR number?
If the PR number is not received then use `*` by default

File: odoo-shippable/scripts/build-image.sh [hunk 3]
```diff
@@ -235,11 +235,22 @@ git_ps1_style(){
...
+    shift 1
+    git fetch -p $REMOTE +refs/pull/*/head:refs/pull/$REMOTE/*
+}
 EOF
 
+# Load .container.profile
+if [ ! -f "~/.container.profile" ]; then
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: `!` is a not `-f` is a exists file
Then you have a `if` here inverted

Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: Thanks!

File: odoo-shippable/scripts/build-image.sh (hunk 2 above)
Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: Yes, now the git_fetch_pr receive a second parameter

**Pull request merged**